tests/unit/test_charm.py

```diff
@@ -98,7 +99,7 @@ def test_relation_joined_when_leader(urlopen_mock, metadata_file):
         )
         harness.begin()
         harness.charm.on.config_changed.emit()
-        harness.model.unit.status == ops.ActiveStatus()
+        assert harness.model.unit.status == ops.ActiveStatus()
```

Comment:
Would it be better if we asserted the `status.name` for consistency with tests above?

Reply:
Not necessarily. The only reason we are checking only the name in the assertion above is that the status contains a message